### PR TITLE
Confluence native poller connector

### PR DESCRIPTION
**Pull request by vu-socprime**
## Proposed Changes

  - add Confluence native poller connector

### CODE REVIEW

File: DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                            "query": "{{graphQueriesTableName}}\n| sort by TimeGenerated desc"
+                        }
+                    ],
+                    "dataTypes": [
+                        {
+                            "name": "{{graphQueriesTableName}}",
+                            "lastDataReceivedQuery": "{{graphQueriesTableName}}\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+                        }
+                    ],
+                    "connectivityCriterias": [
```
Comment (NikTripathi) [inline DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json:56]: It should be connectivityCriteria. Please remove s.

File: DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    "request": {
+                        "apiEndpoint": "https://{{domain}}/wiki/rest/api/audit",
+                        "httpMethod": "Get",
+                        "queryTimeFormat": "yyyy-MM-ddTHH:mm:ssZ",
+                        "startTimeAttributeName": "startDate",
+                        "endTimeAttributeName": "endDate",
+                        "queryWindowInMin": 5
+                    },
+                    "paging": {
+                        "pagingType": "Offset",
```
Comment (NikTripathi) [inline DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json:132]: Value "Offset" is not defined in enum.

**Review by NikTripathi — CHANGES_REQUESTED** [review]

File: DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json (hunk 2 above)
Comment (vu-socprime) [reply DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json:132]: It is a new type of pagination. It was developed recently and provided to us by Microsoft team.

File: DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json (hunk 1 above)
Comment (vu-socprime) [reply DataConnectors/AtlassianConfluenceAudit/ConfluenceNativePollerConnector/azuredeploy_Confluence_native_poller_connector.json]: fixed

**Pull request merged**